flatpak-import.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python2
+
+from argparse import ArgumentParser
+from ConfigParser import ConfigParser
+import logging
+import os
+from sys import exit
+
+import gi
+gi.require_version('OSTree', '1.0')
+from gi.repository import GLib, Gio, OSTree
+
+
+OSTREE_COMMIT_GVARIANT_STRING = "(a{sv}aya(say)sstayay)"
+
+OSTREE_STATIC_DELTA_META_ENTRY_FORMAT = "(uayttay)"
+
+OSTREE_STATIC_DELTA_FALLBACK_FORMAT = "(yaytt)"
+
+OSTREE_STATIC_DELTA_SUPERBLOCK_FORMAT = \
+    "(a{sv}tayay" + OSTREE_COMMIT_GVARIANT_STRING + \
+    "aya" + OSTREE_STATIC_DELTA_META_ENTRY_FORMAT + \
+    "a" + OSTREE_STATIC_DELTA_FALLBACK_FORMAT + \
+    ")"
+
+
+if __name__ == "__main__":
+    aparser = ArgumentParser(
+        description='Import flatpak into a local repository ',
+    )
+    aparser.add_argument('repo',
+                         help='repository name to use')
+    aparser.add_argument('flatpak', help='file to import')
+    aparser.add_argument('-g', '--gpg-homedir',
+                         help='GPG homedir to use when looking for keyrings')
+    aparser.add_argument('-k', '--keyring', help='additional trusted keyring file')
+    aparser.add_argument('-s', '--sign-key', help='GPG key ID to sign the commit with')
+    aparser.add_argument('-v', '--verbose', action='store_true',
+                         help='verbose log output')
+    aparser.add_argument('-d', '--debug', action='store_true',
+                         help='debug log output')
+
+    config = ConfigParser()
+    config.read('flatpak-import.conf')
+    aparser.set_defaults(**dict(config.items('defaults')))
+
+    args = aparser.parse_args()
+
+    if args.debug:
+        logging.basicConfig(level=logging.DEBUG)
+    elif args.verbose:
+        logging.basicConfig(level=logging.INFO)
+    else:
+        logging.basicConfig(level=logging.WARNING)
+
+    ### Mmap the flatpak file and create a GLib.Variant from it
```

Comment:
Everything here and below should move into a function - generally you want the `__name__ == '__main__'` to be as small as possible.